__tests__/validate.test.ts

```diff
@@ -1,26 +1,64 @@
 import * as path from 'path'
 import * as validate from '../src/validate'
 
-test('validates wrapper jars', async () => {
-  const invalidWrapperJars = await validate.findInvalidWrapperJars(
-    path.resolve('.'),
-    2,
-    false,
-    []
+const baseDir = path.resolve('.')
+
+test('succeeds if all found wrapper jars are valid', async () => {
+  const result = await validate.findInvalidWrapperJars(baseDir, 2, false, [
+    'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855'
+  ])
+
+  expect(result.isValid()).toBe(true)
+
+  expect(result.toDisplayString()).toBe(
+    '✓ Found known Gradle Wrapper JAR files\n' +
```

Comment:
Should this be `Found known Gradle Wrapper JAR files:\n` with a `:`?

Reply:
Done